tests/functional/u/used/used_before_assignment_conditional.py

```diff
@@ -1,6 +1,6 @@
 """used-before-assignment cases involving IF conditions"""
 
-# pylint: disable=consider-using-augmented-assign
+# pylint: disable=consider-using-augmented-assign\
```

Comment:
```suggestion
# pylint: disable=consider-using-augmented-assign
```

Reply:
what is the suggestion?

Reply:
I added a line with enum comparison could you please verify this is what you meant?

Reply:
No just a mistake probably because I added it as a core checker and then converted it to a plug-in checker. Deleting all the --disable in all tests, maybe I pressed `\` instead of `del` button in the keyboard..